views/place.hbs

```diff
@@ -1,54 +1,64 @@
+{{>header}}
+{{>options}}
 
-<ul class="">
-  <div class="">
-    <a href="/">{{localText.homePage}}</a>
-    <a href="/{{lang}}/places">{{localText.backList}}</a>
+<a class="f6 ma4 w4 tc grow no-underline br-pill ba bw1 pv2 dib mid-gray b gray" href="/{{lang}}/add-place">{{localText.add}}</a>
+<div class="pa4">
+  <div class="overflow-auto">
+    <table class="f6 w-100 mw8 center" cellspacing="0">
+      <thead>
+        <tr class="ma2">
+          <th class="fw6 tl pa3">
+            <img src="{{place.imageUrl}}" alt="place image">
+          </th>
+          <th class="f1 tc dark-text">{{place.local.name}}</th>
+          <th class="fw6 pa3">
+              <a href="/{{../lang}}/edit-place/{{_id}}" class="f6 w4 grow no-underline br-pill ba bw1 pv2 mv2 db mid-gray b gray">{{localText.edit}}</a>
+          <th>
+            <a href="/{{../lang}}/delete-place/{{_id}}" class="f6 w4 tc grow no-underline br-pill ba bw1 ph4 pv2 mv2 pv2 db mid-gray b gray">{{localText.delete}}</a>
+          </th>
+          </th>
+        </tr>
+      </thead>
+      <tbody class="">
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.description}}:</td>
+          <td class="tl light-text f4">{{place.local.description}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.placeAddress}}:</td>
+          <td class="tl light-text f4">{{place.local.address}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.placeHours}}:</td>
+          <td class="tl light-text f4">{{place.local.openingHours}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.placeWebsite}}:</td>
+          <td class="tl light-text f4">{{place.website}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.placePhone}}:</td>
+          <td class="tl light-text f4">{{place.phone}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.placeEmail}}:</td>
+          <td class="tl light-text f4">{{place.email}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.accessibilityOptions}}:</td>
+          <td class="tl light-text f4">{{place.accessibilityOptions}}</td>
+        </tr>
+        <tr class="">
+          <td class="tr light-text f4 b pa3">{{localText.categories}}:</td>
+          <td class="tl light-text f4">{{place.categories}}</td>
+        </tr>
+      </tbody>
+    </table>
   </div>
-  <div>
-    <img src="{{place.imageUrl}}" alt="place image">
+  <div class="f3 b text-turquoise mw8 center">
```

Comment:
don't like this - instead on you should give the table element the class `bb` to give it border bottom and then make a border color class in the css like `b--turquoise` with the right color - then you can change the thickness with `bw1/2/3` on the table element as well

Reply:
In fact maybe also give it `bt` for border top cos that might look good